### PR TITLE
♻️ Updated Slider tokens

### LINKED ISSUE #1256: Conform Slider tokens

### PR DESCRIPTION
**Pull request by pomfrida**
Resolves #1256

### CODE REVIEW

**Comment by pomfrida** [comment]
Getting this annoying and pointless error after eslint automatically adds a new line
Screenshot 2021-04-28 at 15 25 47

File: libraries/core-react/src/components/Slider/Slider.tsx [hunk 1]
```diff
@@ -122,7 +118,7 @@ const Wrapper = styled.div<WrapperProps>`
   &:hover:not([disabled]) {
     ${fakeTrackBgHover}
     &::after {
-      background: ${enabled.track.indicator.hover.color};
+      background: ${track.states.hover.background};
```
Comment (mimarz) [inline libraries/core-react/src/components/Slider/Slider.tsx:121]: This should probably be `indicator.states.hover.background`?

Comment (pomfrida) [reply libraries/core-react/src/components/Slider/Slider.tsx]: Fixed 👍

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**